spec/features/orders_view_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+describe "the order view", type: :feature do
+
+  before :each do
+    @admin = FactoryGirl.create(:admin)
+    @order = FactoryGirl.create(:order)
+    visit login_path
+    fill_in("session_email", with: @admin.email)
+    fill_in("session_password", with: @admin.password)
+    click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.